server/api/custom/graphqlServer.js

```diff
@@ -0,0 +1,236 @@
+import express from "express";
+import GraphQLHTTP from "express-graphql";
+import schema from "./graphqlSchema";
+import axios from "axios";
+
+const app = express();
+const PORT = 8000;
+app.use("/graphql", GraphQLHTTP({
+  schema,
+  graphiql: true,
+  pretty: true
+})
+);
+
+app.get("/api/products", (request, response) => {
+  axios.post(`http://${request.headers.host}/graphql`,
+    {query: "{ products {title _id vendor price inventoryQuantity}}"},
+    {
+      headers: {
+        "Content-Type": "application/json"
+      }
+    })
+    .then((res) => {
+      response.json(res.data);
+    })
+    .catch((error) => {
+      response.send(error);
+    });
+});
+
+app.get("/api/users", (request, response) => {
+  axios.post(`http://${request.headers.host}/graphql`,
+    {query: `
+      {
+        users {
+          userId
+          shopId
+          fullName
+          emails
+          verified
+          createdAt
+        }
+      }`
+    },
+    {
+      headers: {
+        "Content-Type": "application/json"
+      }
+    })
+    .then((res) => {
+      response.json(res.data);
```

Comment:
I do not see you return any status code here and in most of the other endpoints.

Kindly rectify that.